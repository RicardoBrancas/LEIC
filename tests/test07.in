C AAA
A SER 11
A RAE 9
A ULA 14
A YCZ 20
G ULA YCZ
L 1
F SER YCZ
F YCZ SER
F ULA RAE
A UHF 4
G YCZ SER
Q
G UHF YCZ
R ULA YCZ
G SER UHF
F ULA YCZ
P
G UHF ULA
P
G YCZ ULA
F RAE UHF
F ULA SER
I UHF 18
I ULA -4
R ULA YCZ
F ULA SER
L 1
I YCZ -18
F UHF RAE
S YCZ ULA
I UHF 14
F RAE YCZ
F SER YCZ
F RAE ULA
R SER YCZ
F UHF RAE
G YCZ UHF
F ULA SER
A NXM 1
F ULA RAE
S YCZ UHF
F YCZ RAE
S UHF ULA
F SER RAE
C YCZ
F RAE YCZ
A ARS 14
G SER YCZ
V
G YCZ ULA
F SER UHF
F RAE NXM
G ULA NXM
F UHF SER
L 0
G NXM RAE
F YCZ ULA
A PIU 5
R UHF NXM
F NXM ULA
F PIU YCZ
G PIU NXM
N ULA UHF
Q
A ACY 1
R UHF RAE
F UHF NXM
G UHF ARS
S SER NXM
F NXM UHF
R UHF ULA
I ARS -14
G SER NXM
Q
P
F ULA RAE
G ARS PIU
R YCZ RAE
G SER RAE
C ARS
G YCZ PIU
G YCZ ARS
P
F SER RAE
F ACY PIU
L 1
Q
F ACY YCZ
I YCZ -2
A RRD 13
G NXM RRD
G NXM RRD
F UHF ULA
V
F NXM ARS
F YCZ PIU
N UHF RRD
F YCZ NXM
F ACY YCZ
G UHF ARS
N ULA RAE
S NXM UHF
R RAE NXM
F ACY PIU
G RAE SER
C YCZ
G SER ULA
A HFL 16
F ARS SER
A KMN 1
F RRD UHF
A NER 15
I YCZ 7
C ARS
V
L 0
V
L 0
G SER ULA
G RRD RAE
F ARS ULA
A WGA 20
F SER YCZ
R ULA SER
N ACY UHF
A WFI 12
G NER ULA
G YCZ NER
R RRD WFI
G NXM WFI
G NER WFI
G ACY ARS
C KMN
I YCZ 19
F YCZ RAE
N ULA SER
G WFI PIU
F WGA KMN
G ARS YCZ
S RAE ULA
G KMN RRD
L 2
S RRD ACY
G ACY WFI
F ARS RRD
R ARS PIU
R RAE UHF
A JLF 12
S RAE ARS
R NXM ULA
A JMP 15
G RAE SER
F ARS PIU
G ACY UHF
Q
F HFL ARS
P
L 1
F RRD WFI
P
F ULA KMN
G JMP RRD
G RRD ARS
G JLF WFI
G RAE NER
G JLF JMP
G UHF NXM
L 0
P
N UHF KMN
F NXM WFI
N WGA RAE
G HFL WFI
Q
G UHF ARS
L 2
F RAE UHF
F RAE WFI
S WGA YCZ
S WFI ULA
G NXM YCZ
N UHF JLF
F UHF YCZ
S UHF HFL
I NER -9
F WGA JLF
G NER PIU
L 2
A QSY 6
G RRD ULA
G ARS KMN
G ARS JMP
G YCZ UHF
F RRD JMP
G UHF SER
R UHF HFL
A YHW 6
R ULA JMP
N UHF SER
P
F NER YCZ
N KMN WGA
A CEI 13
A SFV 16
X