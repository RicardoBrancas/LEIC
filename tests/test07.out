*Aeroporto AAA inexistente
RAE:9:0:0
SER:11:0:0
ULA:14:1:0
YCZ:20:0:1
Aeroporto com mais ligacoes ULA:2
Aeroporto com mais rotas YCZ:4:4
Aeroporto com mais rotas YCZ:4:4
*Impossivel adicionar voo RT RAE UHF
RAE:9:1:1
SER:11:5:5
UHF:22:2:1
ULA:10:3:6
YCZ:20:5:3
*Capacidade de YCZ inalterada
*Impossivel remover voo RT YCZ ULA
*Impossivel adicionar voo RT SER YCZ
*Impossivel adicionar voo RT RAE ULA
*Impossivel adicionar voo RT ULA SER
*Impossivel adicionar voo RT ULA RAE
*Impossivel adicionar voo RT YCZ RAE
*Impossivel remover voo RT UHF ULA
*Impossivel adicionar voo RT SER RAE
*Impossivel adicionar voo RT RAE YCZ
*Impossivel adicionar voo SER YCZ
Voo mais popular SER:ULA:2
*Impossivel adicionar voo YCZ ULA
*Impossivel adicionar voo RT RAE NXM
SER:11:5:4
RAE:9:3:3
ULA:10:4:4
YCZ:20:0:0
UHF:36:5:5
NXM:1:0:1
ARS:14:0:0
*Impossivel adicionar voo NXM RAE
*Impossivel adicionar voo RT YCZ ULA
*Impossivel remover voo UHF NXM
*Impossivel adicionar voo RT NXM ULA
*Impossivel adicionar voo RT PIU YCZ
*Impossivel adicionar voo PIU NXM
Voos entre cidades ULA:UHF:0:1
Aeroporto com mais ligacoes ULA:4
*Impossivel adicionar voo RT UHF NXM
*Impossivel remover voo RT SER NXM
*Impossivel adicionar voo RT NXM UHF
*Capacidade de ARS inalterada
*Impossivel adicionar voo SER NXM
Aeroporto com mais ligacoes ULA:3
Aeroporto com mais rotas SER:5:4
*Impossivel remover voo YCZ RAE
*Impossivel adicionar voo YCZ PIU
*Impossivel adicionar voo YCZ ARS
Aeroporto com mais rotas SER:6:4
*Impossivel adicionar voo RT SER RAE
*Impossivel adicionar voo RT ACY PIU
ACY:1:0:0
ARS:14:0:0
NXM:1:0:1
PIU:5:0:0
RAE:9:4:4
SER:11:6:4
UHF:36:3:5
ULA:10:5:4
YCZ:20:0:0
Aeroporto com mais ligacoes SER:3
*Impossivel adicionar voo RT ACY YCZ
*Capacidade de YCZ inalterada
*Impossivel adicionar voo NXM RRD
*Impossivel adicionar voo NXM RRD
*Impossivel adicionar voo RT UHF ULA
Voo mais popular SER:UHF:3
*Impossivel adicionar voo RT NXM ARS
*Impossivel adicionar voo RT YCZ PIU
Voos entre cidades UHF:RRD:0:0
*Impossivel adicionar voo RT YCZ NXM
*Impossivel adicionar voo RT ACY YCZ
*Impossivel adicionar voo UHF ARS
Voos entre cidades ULA:RAE:2:2
*Impossivel remover voo RT NXM UHF
*Impossivel remover voo RAE NXM
*Impossivel adicionar voo RT ACY PIU
*Impossivel adicionar voo SER ULA
*Impossivel adicionar voo RT ARS SER
*Capacidade de YCZ inalterada
Voo mais popular SER:UHF:3
SER:11:6:5
RAE:9:5:4
ULA:10:5:4
YCZ:20:0:0
UHF:36:4:6
NXM:1:0:1
ARS:14:0:0
PIU:5:0:0
ACY:1:0:0
RRD:13:1:1
HFL:16:0:0
KMN:1:0:0
NER:15:0:0
Voo mais popular SER:UHF:3
SER:11:6:5
RAE:9:5:4
ULA:10:5:4
YCZ:20:0:0
UHF:36:4:6
NXM:1:0:1
ARS:14:0:0
PIU:5:0:0
ACY:1:0:0
RRD:13:1:1
HFL:16:0:0
KMN:1:0:0
NER:15:0:0
*Impossivel adicionar voo SER ULA
*Impossivel adicionar voo RRD RAE
*Impossivel adicionar voo RT ARS ULA
*Impossivel adicionar voo RT SER YCZ
Voos entre cidades ACY:UHF:0:0
*Impossivel adicionar voo YCZ NER
*Impossivel remover voo RRD WFI
*Impossivel adicionar voo NXM WFI
*Impossivel adicionar voo ACY ARS
*Capacidade de YCZ inalterada
*Impossivel adicionar voo RT YCZ RAE
Voos entre cidades ULA:SER:1:2
*Impossivel adicionar voo RT WGA KMN
*Impossivel adicionar voo ARS YCZ
*Impossivel adicionar voo KMN RRD
0:6
1:2
2:3
7:2
10:2
*Impossivel remover voo RT RRD ACY
*Impossivel adicionar voo RT ARS RRD
*Impossivel remover voo ARS PIU
*Impossivel remover voo RT RAE ARS
*Impossivel remover voo NXM ULA
*Impossivel adicionar voo RT ARS PIU
*Impossivel adicionar voo ACY UHF
Aeroporto com mais ligacoes ULA:4
*Impossivel adicionar voo RT HFL ARS
Aeroporto com mais rotas SER:6:5
ACY:1:1:0
ARS:14:0:0
HFL:16:0:0
JLF:12:0:0
JMP:15:0:0
KMN:1:0:0
NER:15:2:0
NXM:1:0:1
PIU:5:0:1
RAE:9:4:3
RRD:13:1:1
SER:11:6:5
UHF:36:4:5
ULA:10:3:4
WFI:12:1:2
WGA:20:0:0
YCZ:20:0:0
Aeroporto com mais rotas SER:6:5
*Impossivel adicionar voo RT ULA KMN
*Impossivel adicionar voo RRD ARS
*Impossivel adicionar voo UHF NXM
SER:11:6:5
RAE:9:5:3
ULA:10:3:4
YCZ:20:0:0
UHF:36:4:5
NXM:1:0:1
ARS:14:0:0
PIU:5:0:1
ACY:1:1:0
RRD:13:2:3
HFL:16:0:0
KMN:1:0:0
NER:15:2:1
WGA:20:0:0
WFI:12:2:4
JLF:12:2:0
JMP:15:1:1
Aeroporto com mais rotas SER:6:5
Voos entre cidades UHF:KMN:0:0
*Impossivel adicionar voo RT NXM WFI
Voos entre cidades WGA:RAE:0:0
Aeroporto com mais ligacoes WFI:6
*Impossivel adicionar voo UHF ARS
0:4
1:4
2:2
3:1
5:1
7:2
8:1
9:1
11:1
*Impossivel adicionar voo RT RAE UHF
*Impossivel adicionar voo RT RAE WFI
*Impossivel remover voo RT WGA YCZ
*Impossivel remover voo RT WFI ULA
*Impossivel adicionar voo NXM YCZ
Voos entre cidades UHF:JLF:0:0
*Impossivel adicionar voo RT UHF YCZ
*Impossivel remover voo RT UHF HFL
0:3
1:3
2:3
4:2
5:1
7:2
8:1
9:1
11:1
*Impossivel adicionar voo ARS KMN
*Impossivel adicionar voo ARS JMP
*Impossivel adicionar voo YCZ UHF
*Impossivel adicionar voo UHF SER
*Impossivel remover voo UHF HFL
*Impossivel remover voo ULA JMP
Voos entre cidades UHF:SER:2:3
Aeroporto com mais rotas SER:6:5
*Impossivel adicionar voo RT NER YCZ
Voos entre cidades KMN:WGA:0:0
35:21
